src/main/java/www/purple/mixxy/controllers/ComicController.java

```diff
@@ -17,48 +23,85 @@
 public class ComicController {
 
 	@Inject
-    private ComicDao comicDao;
-	
-  public Result remix() {
-    return Results.TODO();
-  }
-
-  public Result edit() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result newWork() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result likes() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result remixes() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result root() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result parent() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result delete() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result update() {
-    return Results.TODO();
-  }
+	private ComicDao comicDao;
+
+	@SuppressWarnings("PMD.DataflowAnomalyAnalysis")
+	public Result remix() {
+		return Results.TODO();
+	}
+
+	public Result edit() {
+		return Results.TODO();
+	}
+
+	public Result comicShow(@PathParam("id") Long id) {
```

Comment:
Save the slugged title as part of the `Comic`, query by it and its author.  [This](https://github.com/slugify/slugify) looks pretty simple to use for slugifying the title.  Maybe append the ID (in a shortened form, i.e. a higher base than 10) to the URL if you want a quick and easy way to avoid duplicates.